# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ewl/compile.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ewl/src/engines/evas_fb/ewl_engine_evas_fb.c.vanilla	2008-10-28 18:17:00.000000000 +0100
+++ ewl/src/engines/evas_fb/ewl_engine_evas_fb.c	2008-10-28 18:17:28.000000000 +0100
@@ -17,7 +17,7 @@
 static Ecore_Event_Handler *ee_mouse_up_handler = NULL;
 
 static void ee_canvas_setup(Ewl_Window *win, int debug);
-static int ee_init(Ewl_Engine *engine);
+static int ee_init(Ewl_Engine *engine, int *argc, char ** argv);
 static void ee_shutdown(Ewl_Engine *engine);
 
 static void *canvas_funcs[EWL_ENGINE_CANVAS_MAX] =
